lib/plugins/aws/lib/updateStack.js

```diff
@@ -131,15 +131,14 @@ module.exports = {
       params.RollbackConfiguration = this.serverless.service.provider.rollbackConfiguration;
     }
 
-    return this.provider
-      .request('CloudFormation', 'updateStack', params)
-      .then((cfData) => this.monitorStack('update', cfData))
-      .catch((e) => {
-        if (e.message === NO_UPDATE_MESSAGE) {
-          return BbPromise.resolve();
-        }
+    try {
+      const cfData = await this.provider.request('CloudFormation', 'updateStack', params);
+      await this.monitorStack('update', cfData);
+    } catch (e) {
+      if (!e.message.includes(NO_UPDATE_MESSAGE)) {
```

Comment:
I think this error may happen only on `CloudFormation.updateStack` call right?

In such case, I'd wrap with try/catch only `provider.request` invocation.

Reply:
good catch - that's correct :+1: